### PR TITLE
Clarify v2 API testing for podman vs docker clients

### PR DESCRIPTION
**Pull request by cevich**
Fixes: #13273

### CODE REVIEW

**Review by jwhonce — CHANGES_REQUESTED** [review]
Hold on Ghost Rider, while the tests could be better laid out -- python/docker tests use DockerClient **_NOT_** PodmanClient. Given discussions in the group we may want to extend these tests not remove them.

**Comment by cevich** [comment]
> Hold on Ghost Rider,

That's what I half-suspected, thanks for clarifying.  Okay, let me re-push a new version of this that tries to make the situation more clear.  Along those lines, is there something that can be done in the actual tests to show which client is being used?  I'm assuming our setup is correct...but I'd prefer not to assume if I can be told for sure.

File: contrib/cirrus/runner.sh [hunk 1]
```diff
@@ -55,6 +55,7 @@ function _run_unit() {
 }
 
 function _run_apiv2() {
+    warn "Testing podman API with podman client"
```
Comment (edsantiago) [inline contrib/cirrus/runner.sh:58]: Even looking at these additions right next to each other, my eye completely missed the difference. Could you all-caps PODMAN and DOCKER, perhaps even asteriskize them, to make it super-duper clear to someone looking at logs?

Comment (cevich) [reply contrib/cirrus/runner.sh]: actually that's a good idea...

**Comment by cevich** [comment]
Okay there, that should be much better.  Now there's always a message or warning visible no matter if the python-traceback occupies an 10-screenfulls or not :smile:

File: contrib/cirrus/runner.sh [hunk 2]
```diff
@@ -96,8 +100,12 @@ function _run_bindings() {
 }
 
 function _run_docker-py() {
+    m="Testing of API was performed using the **DOCKER** client"
+    warn "$m"
     source venv/bin/activate
-    make run-docker-py-tests
+    if ! make run-docker-py-tests; then
```
Comment (cevich) [inline contrib/cirrus/runner.sh:106]: @edsantiago should this go through the logformatter as well?

Comment (edsantiago) [reply contrib/cirrus/runner.sh:106]: No idea. Test is running, I'll look at results once it finishes and let you know.

Comment (edsantiago) [reply contrib/cirrus/runner.sh:106]: Update: nope, doesn't look like something logformatter will handle right now. (Maybe the one I did last week uses `test -v`, and this one has no `-v`?) Anyway, this LGTM, thank you!

Comment (cevich) [reply contrib/cirrus/runner.sh:106]: Great, thanks.

**Comment by cevich** [comment]
K, this should be good-to-go.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**